packages/frontend-2/components/project/page/models/NewDialog.vue

```diff
@@ -75,4 +77,22 @@ watch(
     }
   }
 )
+
+const dialogButtons = [
+  {
+    text: 'Cancel',
+    props: { color: 'secondary', fullWidth: true, outline: true },
+    onClick: () => {
+      openState.value = false
+    }
+  },
+  {
+    text: 'Create',
+    props: { color: 'primary', fullWidth: true },
+    onClick: () => {
+      onSubmit()
+    },
+    disabled: anyMutationsLoading.value
```

Comment:
dialogButtons isn't reactive, so you're only reading the value of `anyMutationsLoading` once during initialization. Check all of the button definitions in all files and check if they have the same problem. You can probably just wrap them all with a computed to ensure proper reactivity